packages/hierarchies-react/src/presentation-hierarchies-react/UseTree.ts

```diff
@@ -88,14 +87,20 @@ interface GetFilteredPathsProps {
   imodelAccess: IModelAccess;
 }
 
+/** @beta */
+type HierarchyNodeFiltering = Required<HierarchyNode>["filtering"];
+
+/** @beta */
+type FilteredPaths = Required<HierarchyNodeFiltering>["filteredChildrenIdentifierPaths"];
```

Comment:
I think it's incorrect to take this type from `HierarchyNode` - we should take it from `HierarchyProviderProps`. Also, I'd prefer the type to be defined without an intermediate one:


```suggestion
/** @beta */
type HierarchyFilteringPaths = NonNullable<NonNullable<HierarchyProviderProps["filtering"]>["paths"]>;
```